src/roles/netdata/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+
+## If you've installed a custom cert, you can override the self-signed default
+ssl_certificate_file: /etc/haproxy/certs/meza.pem
```

Comment:
Remove `src/roles/netdata/defaults/main.yml` and put the folling into `config/defaults.yml`.

```yaml
ssl_certificate_file: /etc/haproxy/certs/meza.pem
m_install_netdata: True
```

Putting everything into the main `defaults.yml` is just simpler for Meza. Otherwise you're hunting around for what is declaring a particular setting. It's not good, clean Ansible, but Meza uses Ansible differently than it's typically used.